README.md

```diff
@@ -14,50 +14,49 @@
+- Pushing upstream Fedora to harden the build for all Fedora users, including Secureblue users ([for example, by enabling CFI](https://bugzilla.redhat.com/show_bug.cgi?id=2252874)).
+- Reduce the `sudo` timeout to 1 minute.
+- Disable passwordless `sudo` for `rpm-ostree install` <sup>[why?](https://github.com/rohanssrao/silverblue-privesc).
+- Brute force protection by locking user accounts for 24 hours after 50 failed login attempts, hardened password encryption and password quality suggestions.
+- Installing `usbguard` and `bubblejail`.
+- Set `opportunistic` DNSSEC and `DNSOverTLS` for `systemd-resolved`.
+- Configure `chronyd` to use Network Time Security (NTS). <sup>[using chrony config from GrapheneOS](https://github.com/GrapheneOS/infrastructure/blob/main/chrony.conf)</sup>
+- Disable KDE GHNS by default. <sup>[why?](https://blog.davidedmundson.co.uk/blog/kde-store-content/)</sup>
+- Use HTTPS for all RPM mirrors.
+- Set all default container policies to `reject`, `signedBy`, or `sigstoreSigned`.
+- Remove SUID-root from [numerous binaries](https://github.com/secureblue/secureblue/blob/live/config/scripts/removesuid.sh) and replace functionality [using capabilities](https://github.com/secureblue/secureblue/blob/staging/config/files/usr/bin/setcapsforunsuidbinaries).
+- (Non-userns variants) Disabling unprivileged user namespaces.
+- (Non-userns variants) Replacing `bubblewrap` with `bubblewrap-suid` so Flatpak can be used without unprivileged user namespaces.
+
+## Why Fedora ?
+
+[Fedora](https://getfedora.org) is one of the few distributions that ships with [SELinux](https://selinuxproject.org/page/Main_Page) and associated tooling built-in and enabled **by default**. This makes it advantageous as a starting point for building a hardened system. However, out of the box it's **lacking** hardening in numerous other areas. This project's goal is to improve on that __significantly.__
 
 For more info on uBlue and BlueBuild, check out the [uBlue homepage](https://universal-blue.org/) and the [BlueBuild homepage](https://blue-build.org/).
 
 ## Customization
 
-If you want to add your own customizations on top of secureblue, you are advised strongly against forking. Instead, create a repo for your own image by using the [BlueBuild template](https://github.com/blue-build/template), then change your `base-image` to a secureblue image. This will allow you to apply your customizations to secureblue in a concise and maintainable way, without the need to constantly sync with upstream. 
+*If* you want to add your own customizations on top of Secureblue, you are advised **strongly** against forking. Instead, create a repo for your own image by using the [BlueBuild template](https://github.com/blue-build/template), then change your `base-image` to a Secureblue image. This will allow you to apply your customizations to Secureblue in a concise and maintainable way, without the need to constantly sync with upstream. 
```

Comment:
remove these changes. secureblueis lowercase and the emphasis doesn't make sense